itests/paych_api_test.go

```diff
@@ -55,6 +55,15 @@ func TestPaymentChannelsAPI(t *testing.T) {
 	bms := ens.BeginMiningMustPost(blockTime)
 	bm := bms[0]
 
+	waitRecvInSync := func() {
+		// paymentCreator is the block miner, in some cases paymentReceiver may fall behind, so we wait for it to catch up
+
+		head, err := paymentReceiver.ChainHead(ctx)
+		require.NoError(t, err)
+
+		paymentReceiver.WaitTillChain(ctx, kit.HeightAtLeast(head.Height()))
```

Comment:
I'd even suggest doing head + 5 here, just in case of minor reorgs.

Reply:
There's one miner in that test, can't have reorgs